src/fides/api/service/privacy_request/email_batch_service.py

```diff
@@ -41,6 +41,7 @@ def send_email_batch(self: DatabaseTask) -> EmailExitState:
         privacy_requests: Query = (
             session.query(PrivacyRequest)
             .filter(PrivacyRequest.status == PrivacyRequestStatus.awaiting_email_send)
+            .filter(PrivacyRequest.deleted_at.is_(None))
             .order_by(PrivacyRequest.created_at.asc())  # oldest first
```

Comment:
again, there were no specfic requirements here, but I think it makes sense not to send emails for deleted requests

Reply:
smart